### DISCUSSION TITLE
DAI 2023-2024 - My first IntelliJ + Maven project - @RemyBlr

### DISCUSSION THREAD

**Discussion by RemyBlr** [heig-vd-dai-course/heig-vd-dai-course#39] [Show and tell]
https://github.com/RemyBlr/intellij-and-maven

**Comment by ludelafo** [comment]
Thank you!

All good on my side!

Feel free to fill the poll in the GitHub Discussion regarding this chapter: https://github.com/orgs/heig-vd-dai-course/discussions/3.

You can do the same with the chapter Git, GitHub and Markdown: https://github.com/orgs/heig-vd-dai-course/discussions/2.
